pygotham/frontend/__init__.py

```diff
@@ -23,6 +24,7 @@ def create_app(settings_override=None):
 
     assets = Environment(app)
     Foundation(app)
+    Sentry(app)
```

Comment:
Should we restrict this to only non-debug apps or only if a DSN is available? Does Sentry behave well when the DSN is not provided in the environment?